components/dashboard/dashboard.go

```diff
@@ -209,7 +210,7 @@ func (d *Dashboard) ReconcileComponent(cli client.Client, owner metav1.Object, d
 			"<section-title>":       "OpenShift Managed Services",
 		})
 		if err != nil {
-			return fmt.Errorf("Error replacing with correct dashboard url for ConsoleLink: %v", err)
+			return errors.Wrap(err, "failed replacing with correct dashboard url for ConsoleLink: %v")
```

Comment:
do you want to change all the components with this or only dashbaord?

Reply:
Good hint. I'll check the others.

Reply:
@zdtsw this is now reworked using `fmt.Errorf` and `%w` directive, which essentially does the same.